exerciseInJS.html

```diff
@@ -3,48 +3,146 @@
-      <p>
-        <input id="appt-time" type="text" step="2" placeholder="00:00:00"> Hours to Decimal
+    <section class="text">
+      <div class="container-fluid">
+        <div class="container">
+          <h1 id="hoursToD">Hours to Decimal</h1>
+        </div>
+      </div>
+      <div class="container">
+        <h3>Description</h1>
+        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+        tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+        quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+        consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
+        cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
+        proident, sunt in culpa qui officia deserunt mollit anim id est laborum. </p>
+      </div>
+      <div class="container">
+        <input id="appt-time" type="text" step="2" placeholder="00:00:00">  Hours to Decimal 
         <input type="text" id="decimal" placeholder="0.0"> 
-      </p>
+      </div>  
     </section>
 
-    <section>
-      <h1>Stop Watch</h1>
-      <div id="timer">
-        <span id="cronometro">00:00:00</span>    ms  
-        <br>
-        <button type="button" id="stopWatch">Start</button>
-        <br>
-        <div id="times_result" style="background-color: yellowgreen; ">
+    <section class="text">
+      <div class="container-fluid">
+        <div class="container">
+          <h1 id="stoped">Stop Watch</h1>
+        </div>
       </div>
-
+      <div class="container">
+        <h3>Description</h1>
+        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+        tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+        quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+        consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
+        cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
+        proident, sunt in culpa qui officia deserunt mollit anim id est laborum. </p>
+      </div>
+      <div class="container">
+        <div id="timer" class="col-3">
+          <span id="cronometro">00:00:00</span> <button type="button" id="stopWatch" class="col-4">Start</button>     
+          <br>
+          <div id="times_result" class="result"></div>
+        </div>
     </section>
     
+    <section class="ft">
+        <footer>
+          <ul class="content">
+            <li class="ftItem"><a href="#">Contact</a></li>
+            <li class="ftItem"><a href="#">Questions</a></li>
+            <li class="ftItem"><a href="#">Terms and conditions</a></li>
+            <li class="ftItem"><a href="#">Students</a></li>
+            <li class="ftItem"><a href="#">About us</a></li>
+            <li class="ftItem"><a href="#">Privacy</a></li>
+            <li class="ftItem"><a href="#">News</a></li>
+          </ul>
+      </footer>
```

Comment:
Indentation should be 2 spaces.

Reply:
I don't know how to correct it.
I see 2 spaces indentation in my file

Reply:
Make sure your code editor (IDE) is set to use spaces rather than tabs.
<img width="396" alt="Screen Shot 2021-10-15 at 11 42 59" src="https://user-images.githubusercontent.com/3155325/137523329-78a63b10-b53d-48b9-abc0-d6811b0543a7.png">